ivy/functional/frontends/torch/tensor_functions.py

```diff
@@ -14,8 +15,9 @@ def is_tensor(obj):
 # def is_storage(obj):
 # 	return ivy.is_storage(obj)
 
-# def is_complex(obj):
-# 	return ivy.is_complex(obj)
+def is_complex(obj):
+    if ivy.dtype(obj) == complex64 or ivy.dtype(obj) == complex128 or ivy.dtype(obj) == complex192:
```

Comment:
I would actually cast to an Ivy.array here and then compare using Ivy dtypes. That would make the most sense, then return boolean value if it is complex or not :)